ddtrace/contrib/aiohttp/patch.py

```diff
@@ -46,13 +48,19 @@ def __init__(self, obj, pin):
 
     async def connect(self, req, *args, **kwargs):
         pin = Pin.get_from(self)
-        with pin.tracer.trace("%s.connect" % self.__class__.__name__):
+        with pin.tracer.trace("%s.connect" % self.__class__.__name__) as span:
+            # set component tag equal to name of integration
+            span.set_tag("component", config.aiohttp.integration_name)
```

Comment:
Can the additions of component tag be pulled out?

Reply:
yea will add this to another PR